### PR TITLE
Minor correction in docs

### PR DESCRIPTION
**Pull request by amalkurup89**
**What this PR does / why we need it**: Documentation correction

**Which issue(s) this PR fixes**: Minor grammatical correction

### CODE REVIEW

File: docs/sources/installation/docker.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This page also contains important information about [migrating from earlier Dock
...
 
-This variant is highly recommended when security and final image size being as small as possible is desired. The main caveat to note is that it uses [musl libc](http://www.musl-libc.org) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most software doesn't have an issue with this, so this variant is usually a very safe choice.
+This variant is highly recommended when security and final image size being as small as possible is desired. The main caveat to note is that it uses [musl libc](http://www.musl-libc.org) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most softwares don't have an issue with this, so this variant is usually a very safe choice.
```
Comment (achatterjee-grafana) [inline docs/sources/installation/docker.md:24]: One change:
"softwares" should be software.

**Review by achatterjee-grafana — COMMENTED** [review]
Requested a change.

File: docs/sources/installation/docker.md (hunk 1 above)
Comment (amalkurup89) [reply docs/sources/installation/docker.md]: Committed the review recommendation

**Review by achatterjee-grafana — APPROVED** [review]

**Comment by achatterjee-grafana** [comment]
Doc change is merged.

**Comment by amalkurup89** [comment]
@achatterjee-grafana I guess changes are not merged and PR is closed rather. Can you please merge it instead

**Comment by achatterjee-grafana** [comment]
Re-open

**Pull request merged**